tests/core/pyspec/eth2spec/test/helpers/fork_choice.py

```diff
@@ -227,3 +237,21 @@ def apply_next_slots_with_attestations(spec,
     assert store.block_states[block_root].hash_tree_root() == post_state.hash_tree_root()
 
     return post_state, store, last_signed_block
+
+
+def prepare_empty_pow_block(spec, rng=Random(3131)):
+    return spec.PowBlock(
+        block_hash=spec.Hash32(spec.hash(rng.randbytes(32))),
+        parent_hash=spec.Hash32(spec.hash(rng.randbytes(32))),
```

Comment:
@zilm13 
```
        return spec.PowBlock(
>           block_hash=spec.Hash32(spec.hash(rng.randbytes(32))),
            parent_hash=spec.Hash32(spec.hash(rng.randbytes(32))),
            total_difficulty=uint256(0),
            difficulty=uint256(0)
        )
E       AttributeError: 'Random' object has no attribute 'randbytes
```
Our CI environment is Python3.8 and `random.randbytes` was introduced since Python3.9.
[StackOverflow says](https://stackoverflow.com/questions/5495492/random-byte-string-in-python) we can replace `rng.randbytes(32))` with `rng.getrandbits(8) for _ in range(32))`:


```suggestion
        block_hash=spec.Hash32(spec.hash(bytearray(rng.getrandbits(8) for _ in range(32)))),
        parent_hash=spec.Hash32(spec.hash(bytearray(rng.getrandbits(8) for _ in range(32)))),
```